src/codecs/resize/options.tsx

```diff
@@ -95,12 +150,22 @@ export default class ResizerOptions extends Component<Props, State> {
             <option value="mitchell">Mitchell</option>
             <option value="catrom">Catmull-Rom</option>
             <option value="triangle">Triangle (bilinear)</option>
+            <option value="hqx">HQX</option>
```

Comment:
```suggestion
            <option value="hqx">hqx (pixel art)</option>
```

It seems stylized as "hqx" lowercase.